### PR TITLE
Release TKML v4.0.6

### PR DESCRIPTION
**Pull request by jeremyfowers**
Turnkey-LLM:
- (@amd-pworfolk) `oga-load` tool will now use OGA model_builder to automatically create ONNX files for supported CPU and iGPU checkpoints. Manual download of ONNX files is no longer required.
- (@amd-pworfolk, @jeremyfowers) Improved OGA documentation for both iGPU/CPU and NPU 
- (@jeremyfowers) bug fix: HF_TOKEN env var is no longer required to download AMD NPU OGA ONNX files
- (@jeremyfowers) bug fix: server `/health` endpoint now works with OGA (sever is now under CI testing as well)
- (@jeremyfowers) bug fix: server `/ws` always sends a `</s>` at the end of the generation stream

### CODE REVIEW

File: docs/ort_genai_igpu.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+onnxruntime-genai (aka OGA) is a new framework created by Microsoft for running ONNX LLMs: https://github.com/microsoft/onnxruntime-genai/tree/main?tab=readme-ov-file
+
+## Installation
+
+To install:
+
+1. `conda create -n oga-igpu python=3.9`
+1. `conda activate oga-igpu`
+1. `pip install -e path/to/genai[oga-igpu]`
```
Comment (ramkrishna2910) [inline docs/ort_genai_igpu.md:11]: This line refers to internal genai repo, should this be `pip install -e .[llm-oga-dml]`

File: docs/ort_genai_npu.md [hunk 2]
```diff
@@ -21,17 +20,13 @@ onnxruntime-genai (aka OGA) is a new framework created by Microsoft for running
...
+    1. `cd %AMD_OGA%\wheels`
     1. `pip install onnxruntime_genai-0.5.0.dev0-cp310-cp310-win_amd64.whl`
     1. `pip install onnxruntime_vitisai-1.20.0-cp310-cp310-win_amd64.whl`
     1. `pip install voe-1.2.0-cp310-cp310-win_amd64.whl`
-1. Ensure you have access to the models on Hungging Face:
-    1. Ensure you can access the models under [quark-quantized-onnx-llms-for-ryzen-ai-13-ea](https://huggingface.co/collections/amd/quark-quantized-onnx-llms-for-ryzen-ai-13-ea-66fc8e24927ec45504381902) on Hugging Face. Models are gated and you may have to request access.
-    1. Create a Hugging Face Access Token [here](https://huggingface.co/settings/tokens). Ensure you select `Read access to contents of all public gated repos you can access` if creating a finegrained token.
-    1. Set your Hugging Face token as an environment variable: `set HF_TOKEN=<your token>`
 1. Install driver
     1. Access the [AMD RyzenAI EA Lounge](https://account.amd.com/en/member/ryzenai-sw-ea.html#tabs-a5e122f973-item-4757898120-tab) and  download `Win24AIDriver.zip` from `Ryzen AI 1.3 Preview Release`.
```
Comment (ramkrishna2910) [inline docs/ort_genai_npu.md:31]:
```suggestion
    1. Access the [AMD RyzenAI EA Lounge](https://account.amd.com/en/member/ryzenai-sw-ea.html#tabs-a5e122f973-item-4757898120-tab) and  download `Win24AIDriver.zip` from `Ryzen AI 1.3 EA Release`.
```

File: docs/ort_genai_npu.md [hunk 3]
```diff
@@ -40,7 +35,7 @@ onnxruntime-genai (aka OGA) is a new framework created by Microsoft for running
 
 ### Runtime
 
-To test basic functionality, point lemonade to any of the models under under [quark-quantized-onnx-llms-for-ryzen-ai-13-ea](https://huggingface.co/collections/amd/quark-quantized-onnx-llms-for-ryzen-ai-13-ea-66fc8e24927ec45504381902):
+To test basic functionality, point lemonade to any of the models under [quark-quantized-onnx-llms-for-ryzen-ai-13-ea](https://huggingface.co/collections/amd/quark-quantized-onnx-llms-for-ryzen-ai-13-ea-66fc8e24927ec45504381902):
```
Comment (ramkrishna2910) [inline docs/ort_genai_npu.md:38]:
```suggestion
To test basic functionality, point lemonade to any of the models under [quark-quantized-onnx-llms-for-ryzen-ai-1.3-ea](https://huggingface.co/collections/amd/quark-quantized-onnx-llms-for-ryzen-ai-13-ea-66fc8e24927ec45504381902):
```

Comment (jeremyfowers) [reply docs/ort_genai_npu.md]: Fixed in https://github.com/onnx/turnkeyml/pull/244/commits/f50505a733fb2305890e10df9c2dd72888b0c2c7

File: docs/ort_genai_npu.md (hunk 2 above)
Comment (jeremyfowers) [reply docs/ort_genai_npu.md:31]: Fixed in https://github.com/onnx/turnkeyml/pull/244/commits/f50505a733fb2305890e10df9c2dd72888b0c2c7

File: docs/ort_genai_igpu.md (hunk 1 above)
Comment (jeremyfowers) [reply docs/ort_genai_igpu.md]: Fixed in https://github.com/onnx/turnkeyml/pull/244/commits/f50505a733fb2305890e10df9c2dd72888b0c2c7

**Review by ramkrishna2910 — APPROVED** [review]
Looks good!

**Pull request merged**